pkg/backup/client.go

```diff
@@ -400,8 +400,6 @@ func (bc *Client) BackupRanges(
 	updateCh glue.Progress,
 ) ([]*kvproto.File, error) {
 	errCh := make(chan error)
-	ctx, cancel := context.WithCancel(ctx)
-	defer cancel()
```

Comment:
Removing these WithCancel changes the semantic. Are you sure this refactoring is correct?

Reply:
Seems it's OK because this `cancel` is used for just canceling pending tasks while any error occurs. But after #402, the same task could be done by `errorgroup`.

Reply:
Yes, `WithCancel` is unnecessary, all contexts derive from the `defaultContext` in package cmd.